scripts/release.sh

```diff
@@ -342,3 +341,14 @@ function branch_release() {
       ${commitish} \
       ${TAG}
 }
+
+function branch_release() {
+    local tags="$(git tag | cut -d 'v' -f2 | cut -d '.' -f1-2 | sort | uniq )"
+    local branches="$( { (git branch -r | grep origin/release-) ; (git branch  | grep release-); } | cut -d '-' -f2 | sort | uniq)"
```

Comment:
You have to call `setup_upstream()` to fetch the remotes first, as they might not be available.

Reply:
If I call setup_upstream(), then I do I have to grep remote branches? Shoudn't setup_upstream bring all the local branches & tags?

Reply:
Yes. And that's exactly why you have to grep them.